Source/ACE.Database/CharacterDatabase.cs

```diff
@@ -277,8 +277,28 @@ public bool IsNameAvailable(string name)
 
                 // Loads the positions into the player object and resets the landlock id
                 LoadCharacterPositions(c);
-                c.Location = c.Positions[PositionType.Location];
-                c.Location.LandblockId = new LandblockId(c.Location.Cell);

```

Comment:
This line is no longer necessary to set `LandblockId`, as the position's `LandblockId` should now be set when changing the Cell or `LandblockId` of the Position.